### ISSUE TITLE
Temporary files are not closed properly

### ISSUE THREAD

**Issue by FabianUken** [Belval/pdf2image#45]
The temporary files are deleted, but not properly closed. This leads to a too many files open error, when processing large amounts of pdf files.

**To Reproduce**
Example Script:
```
from pathlib import Path
import tempfile
import pdf2image


pdf = Path('./test.pdf').read_bytes()

for i in range(5000):
    with tempfile.TemporaryFile() as tmp_file:
        page_images = pdf2image.convert_from_bytes(pdf, output_file=tmp_file)
        [img.close() for img in page_images]

```

See the leaked files during execution:
`ls -l /proc/{{YOUR PID}}/fd`

**Expected behavior**
The files should get closed properly.

**System:**
 - OS: Ubuntu 18.04
 - Environment: virtualenv with python 3.6.7
 

**Possible fix:**
```
def convert_from_bytes(...):
    fh, temp_filename = tempfile.mkstemp()
    try:
        with open(temp_filename, 'wb') as f:
            f.write(pdf_file)
            f.flush()
            return convert_from_path(...)
    finally:
        os.close(fh)
        os.remove(temp_filename)
```

**Comment by Belval** [comment]
That's bad, if I can reproduce you can expect an updated version in a very near future.

**Comment by Belval** [comment]
The issue is now fixed and deployed to Pypi in version 1.4.1.

Thank you for reporting this issue.
